### PR TITLE
use react-fastclick instead of fastclick

### PR DESCRIPTION
**Pull request by martinratinaud**
In order to fix the bug https://github.com/kriasoft/react-starter-kit/issues/1195. 

Thanks to the help of the community, I found out using fastclick was causing problems when trying to use `react-select` on a safari mobile. I made a demo to illustrate this problem/solution here

https://github.com/martinratinaud/react-starter-kit/tree/fix/react-select-demo

with `fastclick` options are not selectable on safari mobile
with `react-fastclick` it works

Thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "query-string": "^4.3.1",
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
+    "react-fastclick": "^3.0.1",
```
Comment (langpavel) [inline package.json]: You add new dependency but you not removed old one.

**Comment by martinratinaud** [comment]
Ooops, sorry about that, here you go I updated the PR

**Comment by frenzzy** [comment]
Considering to [original fastclick docs](https://github.com/ftlabs/fastclick#when-it-isnt-needed), maybe we should remove fastclick at all?

**Comment by martinratinaud** [comment]
@frenzzy I don't get it. What part of the doc are you refering to exactly ?

Thanks

**Comment by langpavel** [comment]
@martinratinaud Slightly extracted what @frenzzy refs to:

## When it isn't needed
 - Desktop browsers.
 - Chrome 32+ on Android with `width=device-width` in the viewport meta tag
 - Chrome (all) on Android with `user-scalable=no`
 - For IE11+, you can use `touch-action: manipulation;` to disable double-tap-to-zoom on certain elements (like links and buttons). For IE10 use `-ms-touch-action: manipulation`

## When it is needed?
- [ ] Apple software/hardware? Don't know
- [ ] Mobile Firefox? Don't know

`react-fastclick` patch React internals in way I really dislike..

**Comment by martinratinaud** [comment]
I tried to remove react-fastclick and could **not** make it work on my iPhone 5s in both safari and chrome so I guess it's not ready yet.

**Comment by shishirarora3** [comment]
Any resolution yet? Should we remove fast-click completely?

**Comment by jorrit** [comment]
@martinratinaud what do you mean? What doesn't work and in which situation?

**Comment by Bamieh** [comment]
talking from personal experience, fastclick is useless with react, we ended up writing a custom solution at the component level, wrapping any clickable we want, something like `react-tappable`.

**Comment by frenzzy** [comment]
Thanks for discussion, [fastclick](https://github.com/ftlabs/fastclick) dependency was removed from RSK in #1305

**Pull request closed without merging**